application/extensions/admin/grid/CLSGridView.php

```diff
@@ -65,19 +65,23 @@ protected function initColumns()
      */
     protected function lsAfterAjaxUpdate(): void
     {
+        $this->afterAjaxUpdate = 'function(id, data){';
         if (isset($this->lsAfterAjaxUpdate)) {
-            $this->afterAjaxUpdate = 'function(id, data){';
             foreach ($this->lsAfterAjaxUpdate as $jsCode) {
                 $this->afterAjaxUpdate .= $jsCode;
             }
-            $this->afterAjaxUpdate .= 'LS.actionDropdown.create();';
             $this->afterAjaxUpdate .= 'LS.rowlink.create();';
-            $this->afterAjaxUpdate .= '}';
-        } else {
-            // trigger action_dropdown() as a default although no lsAfterAjaxUpdate param passed.
-            // this method is useful for preventing action dropdown cut off && overlapped in other browsers like firefox
-            $this->afterAjaxUpdate = 'function(){ LS.actionDropdown.create(); }';
         }
+        // trigger action_dropdown() as a default although no lsAfterAjaxUpdate param passed.
+        // this method is useful for preventing action dropdown cut off && overlapped in other browsers like firefox
+        $this->afterAjaxUpdate .= 'LS.actionDropdown.create();';
+        $this->afterAjaxUpdate .= 'LS.CPDB.bindButtons();';
+        $this->afterAjaxUpdate .= 'LS.CPDB.participantPanel();';
+        // rebind participant action dropdown buttons
+        $this->afterAjaxUpdate .= 'bindListItemclick();';
+        // rebind attribute action dropdown buttons
+        $this->afterAjaxUpdate .= 'LS.CPDB.attributePanel();';
+        $this->afterAjaxUpdate .= '}';
     }
```

Comment:
as already mentioned in https://github.com/LimeSurvey/LimeSurvey/pull/3419 specific functions should not be in the global bbut in the code where the gridview for the attributecontrol is defined